weaver/cli.py

```diff
@@ -1221,7 +1221,10 @@ def jobs(self,
         if sort is not None:
             query["sort"] = sort
         if isinstance(status, str) and status:
-            query["status"] = map_status(status)
+            status = status.split(",")
+        if isinstance(status, list) and status:
+            status = [map_status(_status) for _status in status]
+            query["status"] = ",".join(status)
```

Comment:
Is a list of statuses supported?
I tried with `-S succeeded` and `-S started` successfully, but with `-S succeeded started`, I get an error :
```
{
  "code": "JobInvalidParameter",
  "description": "Job query parameters failed validation.",
  "error": "Invalid",
  "cause": "{'status': '\"succeeded,running\" is not one of dismissed, failed, running, '\n           'accepted, succeeded'}",
  "value": {
    "status": "succeeded,running",
    "process": null,
    "provider": null
  }
}
```
It seems not to crash here, but during the actual `GET` request. Do you get the same problem?

Reply:
I noticed this mornign that the API validator blocked it. Working on it.